src/App.tsx

```diff
@@ -24,7 +26,7 @@ const App = () => {
       errorElement: <NotFound />,
       children: [
         { index: true, element: <Login /> },
-        { path: "login", element: <Login /> },
+        { path: "login", element: <Login/> },
         { path: "register", element: <Register /> },
         { path: "forgetpass", element: <ForgetPassword /> },
```

Comment:
The convention with naming URL paths is to make it small letters, and when having multiple words to separate  them with hyphen as the example below from Next.js page. 
![image](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/cc3e8bb4-ff7e-46d0-89c4-acdc929f1891)

The problem with using camelCase or having multiple words together is that search engines will treat these paths as a single word. If we try to select  it, we will find the following:
![URL Naming](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/e17538c9-a6a7-4d60-910a-077cf5648225)

Reply:
changed done